vignettes/ggPMX-ggplot.Rmd

```diff
@@ -0,0 +1,81 @@
+---
+title: "How to use ggplot2 syntax with ggPMX"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to use ggplot syntax with ggPMX}
+  %\VignetteEngine{knitr::rmarkdown}
```

Comment:
You use `html_vignette` which is fine, but some of the other vignettes are `pdf` vignettes.

Reply:
Hi @mattfidler 
"ggPMX with nlmixr" - html vignette
"ggPMX - User Guide" - pdf vignette
Do you want to use the pdf format for all vignettes or HTML format?

Reply:
I prefer `html` because it can be used more easily in `pkgdown`, I think. @biethbr1  do you have any preference.

Reply:
@baltcir1 do you have any preference.

Reply:
@mattfidler I like pdf because it's good for printing. But indeed, html is convenient for browsing. Can we have both?

Reply:
@baltcir1 perhaps the user manual should stay a pdf.  Otherwise you choose for each (you can knot to anything).

Reply:
Hi @mattfidler, 
I committed changes for ggplot vignette. 
Regarding format - as I understood you are fine to use HTML format for all vignettes, except guide.
This is exactly what we have right now (nlmixr, ggplot -html, guide -pdf).